.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,27 @@
-### 🛠 Purpose
+### Purpose 🚀
+_clear, one sentence description of primary purpose of this PR_
 
----
+###  Notes ✏️
+_details of code change / secondary purposes of this PR_
 
-### ✏️ Notes to Reviewer
+### Updates 📦
+- [ ] MAJOR (breaking) change to _these packages_
+- [ ] MINOR (backward compatible) change to _these packages_ 
+- [ ] PATCH (big fix) change to _these packages_
 
----
+### Storybook 📙
+http://storybooks.highbond-s3.com/paprika/your-branch-name
```

Comment:
The automatic system doesn't work anymore? Will this need to be a manual step now to update this?

Reply:
Yeah... 😞